core/views.py

```diff
@@ -342,6 +351,10 @@ def get_context_data(self: "CommunityDetailView", **kwargs: any) -> dict[str, an
         context["moderators"] = CommunityMember.objects.filter(
             community=community, role=CommunityMember.MODERATOR
         ).select_related("user")
+
+        context["community_avatar_url"] = community.avatar.url if community.avatar else None
```

Comment:
```suggestion
        context["community_avatar_url"] = community.avatar_url
```
Same for background_url please